src/api-types.ts

```diff
@@ -1064,3 +1134,34 @@ export type UpdatePropertySchema =
   | UpdateMultiSelectPropertySchema
   | RenamePropertySchema
   | null
+
+/*
+ * Files
+ */
+
+interface FileBase {
+  type: "file"
+  file: {
+    url: string
+  }
+}
+
+export type File = FileBase & { file: { expiry_time: string } }
+
+export type FileInput = Omit<FileBase, "type"> & { type?: "file" }
```

Comment:
another place to use `PartialBy`?

Reply:
Good catch! I did this one too but somehow it wasn't committed. Just pushed this change.